crowdsec-docs/unversioned/getting_started/installation/kubernetes.mdx

```diff
@@ -19,6 +19,8 @@ Before getting started, it is advised to read the [introduction](/unversioned/ge
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 - [Helm](https://helm.sh/docs/intro/install/)
 
+Even if an installation could be possible without Helm, it's not supported for now.
```

Comment:
"its not documented for now"

as it is supported if they go through and actually learn how to write it themselves.